app/uk/gov/hmrc/incometaxsubscriptioneligibility/config/AppConfig.scala

```diff
@@ -36,4 +37,15 @@ class AppConfig @Inject()(servicesConfig: ServicesConfig) extends FeatureSwitchi
 
   lazy val desEnvironmentHeader: (String, String) = "Environment" -> loadConfig("microservice.services.des.environment")
 
+  def loadConfigFromEnv(key:String): Option[String] = {
+    sys.props.get(key) match{
+      case r@Some(result) if result.nonEmpty => r
+      case _ => Some(servicesConfig.getString(key))
+    }
+  }
+  def loadControlListConfig(param: ControlListParameter): Boolean =
```

Comment:
Because of how you've used this method, it doesn't quite make sense how it's named.

If you changed it to `isEligible`, it'd make the place you're calling it a lot clearer.